www/api/modules/payments/infraestructure/repository/subscription_type/subscription_type_repository.py

```diff
@@ -5,22 +5,24 @@
 from typing import Optional
 
 from ....domain.entity import SubscriptionTypeModel
+# from ....dto.subscription_type_dto import SubscriptionTypeDTO
+# from ....domain.dto import SubscriptionTypeDTO
 
 
 class SubscriptionTypeRepository:
     def __init__(self, db_engine: Engine):
         self.db_engine = db_engine
 
     def get(self,
-            subscription_type_id: str) -> Optional[SubscriptionTypeModel]:
+            subscription_type_id: str):
         with Session(self.db_engine) as session:
             return session.query(SubscriptionTypeModel).filter_by(
                 id=subscription_type_id, deleted_at=None).first()
 
     def get_all(self):
         with Session(self.db_engine) as session:
             return session.execute(text('''
-                SELECT name, payment_cycle, price, currency, is_active
+                SELECT id, name, payment_cycle, price, currency, is_active
                 FROM subscription_types
                 WHERE deleted_at IS NULL AND is_active = TRUE
             ''')).fetchall()
```

Comment:
You can migrate this to use the model, this was written before the model was available

Reply:
i add this solution because using the model give me problems